src/community/index.html

```diff
@@ -123,4 +122,4 @@ <h3>{{ case.name }}</h3>
             {% endif -%}
         {% endfor -%}
     </div>
-</section>
+</section>
```

Comment:
Nit: this change is not needed, and might cause problem when merging w/ upstream later. But I'll let it pass for now.